docs/benchmark.ipynb

```diff
@@ -11,14 +11,25 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## One featurizer to rule them all?\n",
-    "Contrary to many other machine learning domains, _molecular_ featurization (i.e. the process of transforming a molecule into a vector) lacks a good default. It remains unclear how we can effectively capture the richness of molecular data in a unified representation and what works best heavily depends on the nature and constraints of the task you are trying to model. It is therefore good practice to try different featurization schemes: From structural fingerprints, to physico-chemical descriptors and pre-trained embeddings.\n",
+    "Unlike many other machine learning domains, molecular featurization (i.e. the process of transforming a molecule into a vector) lacks a consistently good default. \n",
+    "It is still an open question how to best capture the complexity of molecular data with a unified representation. Which molecular representation works best depends largely on which task you are modeling. To achieve optimal performance, it is wise to experiment with a variety of featurization schemes, from structural fingerprints to physicochemical descriptors and pre-trained embeddings.\n",
     "\n",
-    "## Don't take our word for it\n",
-    "To demonstrate the impact a featurizer can have, we setup two simple benchmarks.\n",
+    "\n",
+    "\n",
+    "## Use cases\n",
+    "\n",
+    "Molecular representation / featurization is an integral part of any molecular modelling workflow and is commonly used for:\n",
+    "\n",
+    "- Search - to find molecules with similar electronic properties, similar structures, or similar biological activity for a target.\n",
+    "- Clustering - to group molecules based on their features and derive hypothesis around the relationship between structure and activity\n",
+    "- Modeling - to build QSAR model for molecular property/activity prediction\n",
```

Comment:
I understand you want to make it less trivial, but I agree with T that if we're targeting ML students interested in DD with these tutorials. I find this too detailed. I wouldn't be able to understand it on my first read through a couple years ago.